presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/JdbcModule.java

```diff
@@ -60,6 +60,8 @@ public void configure(Binder binder)
 
         binder.bind(Key.get(JdbcClient.class, TheJdbcClient.class))
                 .to(Key.get(JdbcClient.class, StatsCollecting.class));
+        binder.bind(Key.get(ConnectionFactory.class, TheConnectionFactory.class))
```

Comment:
I might be late to review, but why `TheConnectionFactory` is needed? I think binding without any annotation would be good enough